contracts/DCAStrategies/DCAStrategies/DCAStrategiesPermissionsHandler.sol

```diff
@@ -71,4 +81,39 @@ abstract contract DCAStrategiesPermissionsHandler is IDCAStrategiesPermissionsHa
   /// @inheritdoc IDCAStrategiesPermissionsHandler
   // TODO: update this after building the new descriptor
   function setNFTDescriptor(IDCAHubPositionDescriptor _descriptor) external override {}
+
+  function _mint(
+    uint256 _id,
+    address _owner,
+    IDCAStrategies.PermissionSet[] calldata _permissions
+  ) internal {
+    _mint(_owner, _id);
```

Comment:
Could we have a `uint256 internal _mintCounter`? That is updated in each mint?

So we can:
1. Avoid passing `_id` to this function
2. The function would look something like:
```
function _mint(address _owner, _permissions) internal returns (uint256 _mintId) {
    _mintId = ++_mintCounter;
    _mint(_owner, _mintId);
    _setPermissions(_mintId, _permissions);
}
3. We can use both `_mintCounter` and `_burnCounter` to calculate the total supply